lib/manager/kustomize/extract.spec.ts

```diff
@@ -344,5 +372,15 @@ describe('manager/kustomize/extract', () => {
         ],
       });
     });
+
+    it('parses helmChart field', () => {
+      const res = extractPackageFile(kustomizeHelmChart);
+      expect(res.deps[0].depType).toBe('HelmChart');
+      expect(res.deps[0].depName).toBe('minecraft');
+      expect(res.deps[0].currentValue).toBe('3.1.3');
+      expect(res.deps[0].registryUrls).toStrictEqual([
+        'https://itzg.github.io/minecraft-server-charts',
+      ]);
```

Comment:
Use single object matcher

Reply:
Sorry, I'm new to javascript, I'm not sure how to do that. I saw other tests are using snapshots. If that's what you meant, how do I create them?

Reply:
```suggestion
      expect(res).toMatchSnapshot({ deps: [
        depType: "HelmChart",
        depName: "minecraft",
        ...
      ]});
```

Something like that

Reply:
Fixed